packages/ffe-datepicker-react/src/calendar/Header.js

```diff
@@ -1,50 +1,139 @@
-                <button
-                    className="ffe-calendar__month-nav ffe-calendar__previous"
-                    onClick={previousMonthHandler}
-                    tabIndex="-1"
-                    title={previousMonthLabel}
-                    type="button"
-                >
-                    <ChevronIkon className="ffe-calendar__icon-prev" />
-                </button>
-                <header
-                    aria-live="assertive"
-                    aria-atomic="true"
-                    className="ffe-calendar__title"
-                    id={`${datepickerId}-title`}
-                >
-                    <div id={`${datepickerId}__month-label`}>
-                        <span className="ffe-calendar__month">{month}</span>
-                        <span className="ffe-calendar__year">{year}</span>
+        <div id={`${datepickerId}-header`} className="ffe-calendar__header">
+            <div
+                id={`${datepickerId}-settings`}
+                className="ffe-calendar__header-container"
+            >
+                <div className="ffe-calendar__header-wrapper">
+                    <div className="ffe-calendar__header-define" />
+                    <div className="ffe-calendar__header-break ffe-calendar__header-break-1" />
+                    <div className="ffe-calendar__header-block" />
+                    <div className="ffe-calendar__header-define" />
+                    <div className="ffe-calendar__header-break ffe-calendar__header-break-2" />
+                    <div className="ffe-calendar__header-block" />
+
+                    <div
+                        id={`${datepickerId}-view1`}
+                        className="ffe-calendar__header-content"
+                    >
+                        <div className="ffe-calendar__header-inner-1">
+                            <div>
+                                <Dropdown
+                                    value={year}
+                                    tabIndex="-1"
+                                    onChange={selectYearHandler}
+                                >
+                                    {renderYearOptions()}
+                                </Dropdown>
+                            </div>
+                            <div>
+                                <MonthPicker
+                                    previousMonthHandler={previousMonthHandler}
+                                    nextMonthHandler={nextMonthHandler}
+                                    month={month}
+                                />
+                            </div>
+                        </div>
+                    </div>
+                    <div
+                        id={`${datepickerId}-view2`}
+                        className="ffe-calendar__header-content"
+                    >
+                        <div className="ffe-calendar__header-inner-2">
+                            <div>
+                                <Dropdown
```

Comment:
En dropdown utan label? Den bør nok i alla fall ha en aria label

Reply:
This should be easy the change. Was not aware it was supposed to have a label.